exporter/exporter.go

```diff
@@ -138,14 +142,50 @@ func main() {
 		log.Fatalf("failed to create database: %v", err)
 	}
 
-	consumers := make([]*Consumer, len(config.Topics))
-	for i, topicConfig := range config.Topics {
-		consumer, err := startConsumer(ctx, config.kafkaBrokers(), tlsConfig, httpClient, topicConfig)
-		if err != nil {
-			log.Fatalf("failed to start consumer: %v", err)
+	consumers := make([]*Consumer, 0)
+	go func() {
+		tries := 0
+		nextTime := (time.Duration(math.Exp2(float64(tries))) * time.Millisecond) + time.Duration(rand.Intn(100))
+		timer := time.NewTimer(nextTime)
+		topics := config.Topics
+
+		for len(topics) > 0 {
```

Comment:
let's retrieve a list of topics here and only start consumers that have topics...